src/widgets/helper/ChannelView.cpp

```diff
@@ -1898,14 +2082,14 @@ void ChannelView::mouseReleaseEvent(QMouseEvent *event)
             {
                 return;
             }
-            else if (hoverLayoutElement->getFlags().has(
-                         MessageElementFlag::Username))
+            if (hoverLayoutElement->getFlags().has(
+                    MessageElementFlag::Username))
             {
                 openTwitchUsercard(this->channel_->getName(),
                                    hoverLayoutElement->getLink().value);
                 return;
             }
-            else if (hoverLayoutElement->getLink().isUrl() == false)
+            if (hoverLayoutElement->getLink().isUrl() == false)
```

Comment:
```suggestion
            if (!hoverLayoutElement->getLink().isUrl())
```

Reply:
I'm not sure I prefer this, I'd rather have a helper function here, or a local variable - the negation is too far away from the verb

Reply:
Didn't see clang-tidy.